setup.py

```diff
@@ -169,7 +169,7 @@ def run(self):
         # Ensure our temporary build directory exists
         try:
             os.makedirs(build_temp)
-        except IOError:
+        except:
```

Comment:
You shouldn't catch anything there. Any error is the real error.
Just test the directory exists instead. The other errors are all fatal.